src/jdk.accessibility/windows/native/include/bridge/AccessBridgeCalls.h

```diff
@@ -31,13 +31,13 @@
  * AccessBridgeCalls.c is available for download from the OpenJDK repository using
  * the following link:
  *
- * http://hg.openjdk.java.net/jdk9/jdk9/jdk/raw-file/tip/src/jdk.accessibility/windows/native/bridge/AccessBridgeCalls.c
+ * http://hg.openjdk.org/jdk9/jdk9/jdk/raw-file/tip/src/jdk.accessibility/windows/native/bridge/AccessBridgeCalls.c
```

Comment:
The domain for hg wasn't changed to openjdk.org.  The original URL is correct.

Reply:
This is correct.
AccessBridge.h is published with the include/header files of the JDK and anyone reading it there can't exactly make use of "../"

Reply:
> This is correct. AccessBridge.h is published with the include/header files of the JDK and anyone reading it there can't exactly make use of "../"

Update to persistent git links.